pkg/health/server/prober.go

```diff
@@ -250,6 +251,13 @@ func (p *prober) httpProbe(node string, ip string, port int) *models.Connectivit
 			scopedLog.WithField("rtt", rtt).Debug("Greeting successful")
 			result.Status = ""
 			result.Latency = rtt.Nanoseconds()
+
+			metrics.Latency.WithLabelValues(
```

Comment:
Those two places are the only places in this file which make requests and store the info about latency. Or am I wrong?

I'm struggling with understanding where we do latency probes to hosts, and where to endpoints.

@joestringer @raybejjani Any hints would be appreciated.

Reply:
I don't think that there's any special logic for hosts vs. endpoints, only for ICMP vs. HTTP probes.